mkdocs.yml

```diff
@@ -1,15 +1,13 @@
 site_name: SimpleLogin Docs
-site_url: https://simplelogin.io/docs/
+site_url: https://simplelogin.io/
```

Comment:
The `site_url` actually corresponds to where the doc is hosted which is https://simplelogin.io/docs/ and not https://simplelogin.io

Reply:
Ho! didn't know that. Trying to make the image in the top left of the docs link to the main site.